nx-X11/programs/Xserver/os/utils.c

```diff
@@ -571,6 +571,7 @@ void UseMsg(void)
     ErrorF("-c                     turns off key-click\n");
     ErrorF("c #                    key-click volume (0-100)\n");
     ErrorF("-cc int                default color visual class\n");
+    ErrorF("-co file               (compatibility option, has no effect)\n");
```

Comment:
we should put the work "deprecated" in here and move it to the very end under a title "deprecated options, kept for compatibility only"

Reply:
there's not really a good place to put those options in the usage message. Putting it at the end will raise more attention than wanted because calling e.g. `nxagent -h` will print the usage and the last lines will stay right before the user's eyes. Putting deprecated options at the top seem wrong, too.